src/PriceOracle.sol

```diff
@@ -17,12 +17,12 @@ contract PriceOracle is AccessControl {
     // Role allowed to update price
     bytes32 public constant PRICE_UPDATER_ROLE = keccak256("PRICE_UPDATER");
 
-    // The current price is the atomic unit
-    uint256 public currentPrice;
-
     // The minimum price allowed
     uint256 public constant minimumPrice = 1024;
 
+    // The current price is the atomic unit.
+    uint256 public currentPrice = minimumPrice;
```

Comment:
Maybe add a require condition within contract that ensures that the `currentPrice` is never less than `minimumPrice`

Reply:
Can you be more specific? I'm not sure where you're proposing to add it. In both functions (`adjustPrice`, `setPrice`) this is already addressed.

Reply:
@significance @istae can you please maybe look into it and if its a bug we can create a new issue